source/tests/system/nidmm_driver_api_tests.cpp

```diff
@@ -244,29 +304,80 @@ TEST_F(NiDmmDriverApiTest, SetViBooleanAttribute_GetViBooleanAttribute_ValueMatc
 TEST_F(NiDmmDriverApiTest, ConfigureMeasurementAbsolute_CompletesSuccessfully)
 {
     ::grpc::ClientContext context;
-    dmm::ConfigureMeasurementAbsoluteRequest request;
-    request.mutable_vi()->set_id(GetSessionId());
-    request.set_measurement_function(dmm::Function::FUNCTION_NIDMM_VAL_DC_VOLTS);
-    request.set_range(10);
-    request.set_resolution_absolute(5.5);
-    dmm::ConfigureMeasurementAbsoluteResponse response;
-    ::grpc::Status status = GetStub()->ConfigureMeasurementAbsolute(&context, request, &response);
-
-    EXPECT_TRUE(status.ok());
-    expect_api_success(response.status());
+  dmm::ConfigureMeasurementAbsoluteRequest request;
+  request.mutable_vi()->set_id(GetSessionId());
+  request.set_measurement_function(dmm::Function::FUNCTION_NIDMM_VAL_DC_VOLTS);
+  request.set_range(10);
+  request.set_resolution_absolute(5.5);
+  dmm::ConfigureMeasurementAbsoluteResponse response;
+  ::grpc::Status status = GetStub()->ConfigureMeasurementAbsolute(&context, request, &response);
+
+  EXPECT_TRUE(status.ok());
+  expect_api_success(response.status());
 }
 
 TEST_F(NiDmmDriverApiTest, ConfigureCurrentSourse_CompletesSuccessfully)
 {
-    ::grpc::ClientContext context;
-    dmm::ConfigureCurrentSourceRequest request;
-    request.mutable_vi()->set_id(GetSessionId());
-    request.set_current_source(0.0001);
-    dmm::ConfigureCurrentSourceResponse response;
-    ::grpc::Status status = GetStub()->ConfigureCurrentSource(&context, request, &response);
-
-    EXPECT_TRUE(status.ok());
-    expect_api_success(response.status());
+  ::grpc::ClientContext context;
+  dmm::ConfigureCurrentSourceRequest request;
+  request.mutable_vi()->set_id(GetSessionId());
+  request.set_current_source(0.0001);
+  dmm::ConfigureCurrentSourceResponse response;
+  ::grpc::Status status = GetStub()->ConfigureCurrentSource(&context, request, &response);
+
+  EXPECT_TRUE(status.ok());
+  expect_api_success(response.status());
+}
+
+TEST_F(NiDmmDriverApiTest, ExportConfiguredCurrentSource_ResetAndImportConfiguration_CurrentSourceConfigurationIsImported)
+{
```

Comment:
This test method should have two empty lines to match with given_when_then format in the test name. I think adding newline before `reset()` call should do.